modules/virt-preparing-container-disk-for-vms.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-create-vms.adoc
+
+[id="virt-preparing-container-disk-for-vms_{context}"]
+= Preparing a container disk for virtual machines
+
+A container disk must be built with a virtual machine image and pushed
+to a container registry before it can be attached to a virtual machine as a containerDisk volume.
+
+.Prerequisites
+
+* You have root or sudo privileges on your local system to install podman.
+
+* The virtual machine image must be either QCOW2 or RAW format.
+
+.Procedure
```

Comment:
Just a thought - how about stating that the user already has `podman` installed? I suspect that documenting how to install packages in RHEL is not too useful.

Reply:
And it's a good thought. In some situations that'd be the way to go but podman installation is not specifically covered in OCP docs - it appears on a case-by-case basis as part of other procedures, much like this one. If you search 'install podman' in the docs set you don't get any useful hits, so any user who is unfamiliar with installing packages will then presumably have to turn to google for the answer. Since this procedure is dependent on podman I think it makes sense for it to be included as part of the procedure itself. But I'll flag this for the reviewer to get their opinion/suggestions too.
I can specify that this step (and prereq) is for RHEL users.